Atomic/mount.py

```diff
@@ -36,10 +36,16 @@
 
 # Module for mounting and unmounting containerized applications.
 
-DMSETUP_PATH='/usr/sbin/dmsetup'
-LSBLK_PATH='/usr/bin/lsblk'
-MOUNT_PATH='/usr/bin/mount'
-FINDMNT_PATH='/usr/bin/findmnt'
+def path_exists(paths):
+    for path in paths:
+        if os.path.exists(path):
+            return path
+    raise ValueError("Unable to find command in {}".format(paths))
```

Comment:
This tool is looking for commands, function should probably be renamed to commands_exists()
Which would clear it up.  Bottom line is the /usr/bin/mount or /bin/mount does not exist.